docs/hub/flair.md

```diff
@@ -0,0 +1,73 @@
+![alt text](https://raw.githubusercontent.com/flairNLP/flair/master/resources/docs/flair_logo_2020_FINAL_day_dpi72.png)
+
+# Using Flair at Hugging Face
+
+[Flair](https://github.com/flairNLP/flair) is:
+
+* **A powerful NLP library.** Flair allows you to apply our state-of-the-art natural language processing (NLP)
+models to your text, such as named entity recognition (NER), sentiment analysis, part-of-speech tagging (PoS),
+  special support for [biomedical data](/resources/docs/HUNFLAIR.md),
+ sense disambiguation and classification, with support for a rapidly growing number of languages.
+
+* **A text embedding library.** Flair has simple interfaces that allow you to use and combine different word and
+document embeddings, including our proposed [Flair embeddings](https://www.aclweb.org/anthology/C18-1139/) and various transformers.
+
+* **A PyTorch NLP framework.** Our framework builds directly on [PyTorch](https://pytorch.org/), making it easy to
+train your own models and experiment with new approaches using Flair embeddings and classes.
+
+Developed by [Humboldt University of Berlin](https://www.informatik.hu-berlin.de/en/forschung-en/gebiete/ml-en/) and friends.
```

Comment:
Can we make this a one- or two-line description instead? Also, I think we shouldn't refer to it as "our" library here.

Reply:
Yeah, I will change it :)

Reply:
It's now a clear and compact two line description :hugs: